x-pack/plugins/maps/common/elasticsearch_util/es_agg_utils.ts

```diff
@@ -63,7 +64,20 @@ export function extractPropertiesFromBucket(
         );
       }
     } else {
-      properties[key] = bucket[key];
+      if (
```

Comment:
I don't think the if check for percentile is needed. Be more generic and just write a case for if `(_.has(bucket[key], 'values'))`. This will be similar to the case at line 50 but instead of `value` it would look for `values`.

Reply:
discussed offline. let's keep the reference to the percentile-constant, so we know why the code-path is here. (similar to the reference to the top-term constant in the other block).